site/app/models/gradeable/GradedGradeable.php

```diff
@@ -39,25 +39,40 @@ class GradedGradeable extends AbstractModel {
 
     /** @prop
      * @var array The late day exceptions indexed by user id */
-    protected $late_day_exceptions = [];
+    protected $late_day_exceptions;
 
     /** @prop
      * @var array<string> The reasons for exceptions indexed by user id */
-    protected $reasons_for_exceptions = [];
+    protected $reasons_for_exceptions;
 
     /** @prop
      * @var bool|null|SimpleGradeOverriddenUser Does this graded gradeable have overridden grades */
     protected $overridden_grades = false;
 
+    /** @prop
+     * @var array<array<int, string>> The active graders for this graded gradeable */
+    protected $active_graders_names;
+
+    /** @prop
+     * @var array<array<int, string>> The active graders for this graded gradeable */
+    protected $active_graders;
+
+    /** @prop
+     * @var array<array<int, string>> The timestamps for the active graders this graded gradeable */
+    protected $active_graders_timestamps;
+
     /**
      * GradedGradeable constructor.
      * @param Core $core
      * @param Gradeable $gradeable The gradeable associated with this grade
      * @param Submitter $submitter The user or team who submitted for this graded gradeable
      * @param array $details Other construction details (indexed by property name)
+     * @param array<array<int, string>> $active_graders The active graders for this graded gradeable
+     * @param array<array<int, string>> $active_graders_timestamps The timestamps for the active graders this graded gradeable
+     * @param array<array<int, string>> $active_graders_names The names for the active graders this graded gradeable
      * @throws \InvalidArgumentException If the provided gradeable or submitter are null
      */
-    public function __construct(Core $core, Gradeable $gradeable, Submitter $submitter, array $details) {
+    public function __construct(Core $core, Gradeable $gradeable, Submitter $submitter, array $details, array $active_graders, array $active_graders_timestamps, array $active_graders_names) {
```

Comment:
What do you think about adding default values for the new parameters?  That way, you won't have to edit any existing code which doesn't use the new parameters.